src/app/[locale]/layout.tsx

```diff
@@ -21,14 +22,24 @@ export default function LocaleLayout({
   params: { locale: any }
 }) {
   if (!getAllAvailableLocales().includes(locale as any)) notFound()
+  const t = useTranslations('current-organisation')
 
   return (
     <html lang={locale} className={`${font.variable}`}>
       <body className="bg-gray-100">
         <AppProvider>
           <Document className="utrecht-theme">
             <Container className="bg-white">
-              <Header />
+              <Header
+                homepage={{
+                  href: '/',
+                  label: t('homepage-label'),
+                }}
+                logo={{
+                  src: '/assets/utrecht.webp',
```

Comment:
Dit ziet er allemaal prima uit. Dit logo moeten we uiteindelijk dan uit de config halen (met in de config het path naar de logo), maar dat is een gemakkelijke fix die we richting het einde kunnen doorvoeren. Hiervoor heeft @bartjkdp ook al een issue aangemaakt (#82)